cruise.umple/src/class/UmpleInternalParser_CodeClass.ump

```diff
@@ -1756,8 +1756,6 @@ private Boolean checkIsDistributed(UmpleInterface uInterface)
       {  // Note: See near clone in UmpleInternalParser_CodeLayout.ump
         String theColor = childToken.getValue("colorValue");
         if(theColor.startsWith("=")) theColor=theColor.substring(1,theColor.length());
-        if(!theColor.startsWith("\"")) theColor= "\""+theColor;
```

Comment:
This PR does not yet solve all cases.

Specifically

```
class X {
  displayColor = "#dfdfdf";
  a;
}
```

(not the space after the equals sign. My suggestion, yes, remove any equals sign. Remove any quotes wherever they are; then use the trim() function to remove spaces before adding the needed quotes.